src/connection_properties.lua

```diff
@@ -29,8 +30,43 @@ function ConnectionProperties:_validate()
     end
 end
 
+--- Get the configured fetchsize in bytes used when fetching query result data.
+--- Configuration property: <code>fetchsize_kib</code>.
+--- Default value: <code>131072</code> = <code>128*1024</code>.
+--- @return number fetchsize in bytes
 function ConnectionProperties:get_fetchsize_bytes() --
     return (self.properties.fetchsize_kib or DEFAULT_FETCHSIZE_KIB) * 1024
 end
 
+--- Get the configured TLS verify mode for connecting to Exasol.
+--- Configuration property: <code>tls_verify</code>.
+--- Default value: <code>none</code>.
+--- Available values: <code>none</code>, <code>peer</code>, <code>client_once</code>, <code>fail_if_no_peer_cert</code>.
```

Comment:
```suggestion
--- Available values: <code>none</code>, <code>peer</code>, <code>client_once</code>, <code>fail_if_no_peer_cert</code>.
```
:+1: hard to maintain, but useful for API users.